README.md

```diff
@@ -37,6 +37,17 @@ DynamicGc.(setup_dynamic_tuning
   });
 ```
 
+We also provide an even simpler way to set up dynamic tuning. The
+`DynamicGc.Config.simple` function takes a threshold heap size and produces a
+config. Below that threshold, the default `space_overhead` value of 120 is used.
+Above the threshold, `space_overhead` is reduced somewhat to 80. With a suitable
```

Comment:
Nit: would the reader of the README here need a reminder of what `space_overhead` measures?

Reply:
At the first reference to `space_overhead` (unchanged in this PR) there is a link to the docs for the `Gc` module. I think that's sufficient.